### DISCUSSION TITLE
Jumping to the next whitespace to ignore consecutive whitespaces? Jumping to empty line?

### DISCUSSION THREAD

**Discussion by rainymatch** [ArturoDent/jump-and-select#9] [Ideas]
I find it very handy that I can jump to the next / previous whitespace. 

However, if there are consecutive whitespaces, like, say, in indented code, I would like to jump to the next whitespace AFTER all these indentation whitespaces.

Same would go for empty lines I guess!

**Comment by ArturoDent** [comment]
I think this would have to behind a setting, perhaps `skip Consecutive/Adjoinging/Multiple WhiteSpace`.  

I wouldn't want one setting for spaces/tabs and one for empty lines so all would have to be controlled by this one setting.

**Comment by rainymatch** [comment]
That sounds good to me.

I think it could be for all characters, too. One global setting for skipping consecutive/adjoining characters.
So it is actually the same story as the point 2. at the other discussion:  https://github.com/ArturoDent/jump-and-select/discussions/10

Let's say I have a case with two consecutive closing parents

```

1|   whatever))
2|
3|
4|  another.something(this(that))
5|
6|  anything
```

Let's say my cursor is at the beginning of the line 1. If I want to jump before the second one one the first line, I would jump before the first one and go rigth with the arrow (rather than jumping twice with jump & select)
If I want to jump before the firts one on the line 4, *I think* I would like to jump just twice, not three times.

Same goes for the empty lines: If I am at line 1 and want to jump to line 5, I would like to jump just twice. 

Can't think of a scenario where I would like to have separate settings for whitespaces and other characters at the moment.
